### PR TITLE
Dprice add manage section

### PR DESCRIPTION
**Pull request by danieltprice**
Add a "manage" section.
https://websitemain62807-dpricemanageusers.gatsbyjs.io/docs/manage/overview/
The purpose of this section is to:
- Provide an overview of the Neon object hierarchy for objects that you work with in the console  (projects, branches, endpoints, etc.)  
- Explain how the objects relate to each other
- Provide introductory information for each object, what it is, etc.
- Provide basic instructions for creating, viewing, deleting, etc.
Some assumptions have been made (for endpoints especially, as the endpoints page is not yet available) 
Note: The ascii diagram is to be replaced by a PP diagram

### CODE REVIEW

File: content/docs/manage/overview.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+## Branches
+
+Data resides in a branch. Each Neon project has a root branch called `main`. You can create child branches from `main` or from previously created branches. A branch can contain multiple databases and users. Tier limits define the number of branches you can create in a project. For more information, see [Branches](../branches).
+
+## Endpoints
+
+An endpoint is the compute instance associated with a branch. A read-write endpoint is created for a project's root branch by default. You can choose whether or not to create an endpoint when creating a branch. To connect to a database that resides in a branch, you must connect via an endpoint that is associated with the branch. Tier limits define the number of endpoints per project and the resources (vCPUs and RAM) available to an endpoint. For more information, see [Endpoints](../endpoints).
```
Comment (stepashka) [inline content/docs/manage/overview.md:54]:
```suggestion
An endpoint is a compute resource associated with a branch. A read-write endpoint is created for a project's root branch by default. You can choose whether or not to create an endpoint when creating a branch. To connect to a database that resides in a branch, you must connect via an endpoint that is associated with the branch. Tier limits define the number of endpoints per project and the resources (vCPUs and RAM) available to an endpoint. For more information, see [Endpoints](../endpoints).
```

not sure about this change, putting this as a siggestion

File: content/docs/manage/api-keys.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+---
+title: API Keys
+enableTableOfContents: true
+redirectFrom:
+  - /docs/get-started-with-neon/using-api-keys
+---
+
+Any action performed in the Neon Console can be performed using the [Neon API](https://neon.tech/api-reference). Using the Neon API requires an API key. This topic describes how to generate, revoke, and use API keys.
```
Comment (raoufchebri) [inline content/docs/manage/api-keys.md:8]:
```suggestion
You can manage your projects, branches, databases, and users using the [Neon API](https://neon.tech/api-reference). Using the Neon API requires an API key. This topic describes how to generate, revoke, and use API keys.
```

I'm not sure of using the `Any action performed`. You can't do things like passwordless login or create an API key

File: content/docs/manage/api-keys.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+<Admonition type="important">
+[Neon API v1](https://console.neon.tech/api/v1) is now deprecated. Support for it will be removed in a future release. Please migrate your applications to [Neon API V2 reference](https://neon.tech/api-reference/v2/). The base URL for the new Neon API is `https://neon.tech/api-reference/v2/`.
+</Admonition>
+
+The following `cURL` example uses the `/projects` endpoint to retrieve projects that belong to your Neon account.
+
+```bash
+curl -X GET -H "Authorization: Bearer $NEON_API_KEY" "accept: application/json"
+"https://neon.tech/api-reference/v2" | jq
```
Comment (raoufchebri) [inline content/docs/manage/api-keys.md:46]:
```suggestion
"https://neon.tech/api-reference/v2"
```
not sure the `| jq` is necessary

File: content/docs/manage/api-keys.md (hunk 2 above)
Comment (danieltprice) [reply content/docs/manage/api-keys.md:8]: OK. I will change that to "most actions".

Comment (raoufchebri) [reply content/docs/manage/api-keys.md:8]: Sounds good

**Review by raoufchebri — APPROVED** [review]

**Review by stepashka — APPROVED** [review]

**Pull request merged**